docs/RELEASE.md

```diff
@@ -41,19 +65,18 @@ Inductively, the public API of our library crates is declared as all public item
 ## Node
 
 The versioning of the Polkadot node is done most of the time by only incrementing the `minor` version. The `major`
-version is only bumped for special releases and the `patch` can be used for an out of band release that fixes some
-critical bug. The node version is not following SemVer. This means that the version doesn't express if there are any
-breaking changes in the CLI interface or similar. The node version is declared in the
-[`NODE_VERSION`](https://paritytech.github.io/polkadot-sdk/master/polkadot_node_primitives/constant.NODE_VERSION.html)
+version is only bumped for special releases and the `patch` is used for a patch release that happens every month and fixes found issues.
+The node version is not following SemVer. This means that the version doesn't express if there are any breaking changes in the CLI interface or similar.
+The node version is declared in the [`NODE_VERSION`](https://paritytech.github.io/polkadot-sdk/master/polkadot_node_primitives/constant.NODE_VERSION.html)
 variable.
 
-## Westend & Rococo
+## Westend
 
 For these networks, in addition to incrementing the `Cargo.toml` version we also increment the `spec_version` and
 sometimes the `transaction_version`. The spec version is also following the node version. Its schema is: `M_mmm_ppp` and
 for example `1_002_000` is the node release `1.2.0`. This versioning has no further meaning, and is only done to map
-from an on chain `spec_version` easily to the release in this repository.  
-The Westend testnet will be updated to a new runtime every two weeks with the latest `nightly` release.
+from an on chain `spec_version` easily to the release in this repository.
```

Comment:
Reading this makes me think that it could be better to change our Westend versioning, but that is for another day.